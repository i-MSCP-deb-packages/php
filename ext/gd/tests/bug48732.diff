001+ Left Bottom: (-1, 41)
001- Left Bottom: (0, 47)