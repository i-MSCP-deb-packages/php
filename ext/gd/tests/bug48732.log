
---- EXPECTED OUTPUT
Left Bottom: (0, 47)
---- ACTUAL OUTPUT
Left Bottom: (-1, 41)
---- FAILED
