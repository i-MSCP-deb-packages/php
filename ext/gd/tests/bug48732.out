Left Bottom: (-1, 41)